exporter/googlecloudpubsubexporter/config.go

```diff
@@ -0,0 +1,51 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package googlecloudpubsubexporter
+
+import (
+	"fmt"
+	"regexp"
+
+	"go.opentelemetry.io/collector/config"
+	"go.opentelemetry.io/collector/exporter/exporterhelper"
+)
+
+var topicMatcher = regexp.MustCompile(`projects/[a-z][a-z0-9\-]*/topics/.*`)
+
+type Config struct {
+	config.ExporterSettings `mapstructure:",squash"`
+	// Timeout for all API calls. If not set, defaults to 12 seconds.
+	exporterhelper.TimeoutSettings `mapstructure:",squash"` // squash ensures fields are correctly decoded in embedded struct.
+	exporterhelper.QueueSettings   `mapstructure:"sending_queue"`
+	exporterhelper.RetrySettings   `mapstructure:"retry_on_failure"`
+	// Google Cloud Project ID where the Pubsub client will connect to
+	ProjectID string `mapstructure:"project"`
+	// User agent that will be used by the Pubsub client to connect to the service
+	UserAgent string `mapstructure:"user_agent"`
+	// Override of the Pubsub endpoint
+	Endpoint string `mapstructure:"endpoint"`
+	// Only has effect if Endpoint is not ""
+	UseInsecure bool `mapstructure:"use_insecure"`
```

Comment:
Be consistent with other places where we use `insecure` https://github.com/open-telemetry/opentelemetry-collector/blob/1f7c6618070197a7818371e747de4e0d61cbc9d9/config/configtls/configtls.go#L54

Reply:
Tried to be consistent with the Google exporter:
https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/aa32f1e9f7cae0093aea16619f6cdbff45d68f88/exporter/googlecloudexporter/config.go#L30

But switch to insecure